WinCompatibilityPack/WinCompatibilityPack.psm1

```diff
@@ -7,6 +7,13 @@ using namespace System.Management.Automation.Runspaces
 
 Set-StrictMode -Version latest
 
+# PSCore 6.0.x has older versions of assemblies from WCP that are not compatible
+if ($PSVersionTable.PSEdition -eq 'Core' -and $PSVersionTable.PSVersion.Major -eq 6 -and
+    $PSVersionTable.PSVersion.Minor -eq 0)
```

Comment:
Jeffrey wants people to use the rmodule capability from non-windows remotely, but we'll resolve that outside of this PR.  Created https://github.com/PowerShell/WindowsPowerShellCompatibilityPack/issues/14

Reply:
The assemblies in this module were gathered by specifying `-r win-x64`. It might not be an issue at all, but better try to see if loading them on a non-windows platform would cause any problem.

Reply:
In the WCP blog table it shows a list of assemblies that can work on non-Windows.  We'll likely have to add more runtimes to this module under netcoreapp20 (like win-x64, linux-x64, etc...) if we want to support WCP on non-Windows